### ISSUE TITLE
Code Inspector - Implicit reference to ActiveWorkbook?

### ISSUE THREAD

**Issue by Pitterling** [rubberduck-vba/Rubberduck#1798]
latest commit

Worksheets(1) is reported as implicit reference to ActiveWorkbook
see also #1157 

```vb
Public Sub AddTempWorkbook()
 Dim defaultSheets As Long
 defaultSheets = Application.SheetsInNewWorkbook            'keep original value
 Application.SheetsInNewWorkbook = 1
 
 Set m_WorkbookTemp = Excel.Workbooks.Add
 m_WorkbookTemp.Worksheets(1).Name = gstrcWsData
 
 Application.SheetsInNewWorkbook = defaultSheets            'reset to default
 
End Sub
```

**Comment by retailcoder** [comment]
The two Excel-specific inspections are currently broken, that's a known but tricky resolver / COM collector issue; shortly put, ...the Excel object model isn't cooperating.

**Comment by Hosch250** [comment]
Fixed with #1882
